packages/common/src/intl/locales/ar/common.json

```diff
@@ -1,6 +1,6 @@
 {
-    "button.ok": "نعم",
-    "button.open-the-menu": "افتح القائمة",
-    "app.admin": "Admin",
-    "app.catalogue": "catalogue"
-}
+  "button.ok": "نعم",
+  "button.open-the-menu": "افتح القائمة",
+  "app.admin": "Admin",
+  "app.catalogue": "catalogue"
+}
```

Comment:
Whitespace changes due to Prettier reformatting on save -- shouldn't change again.